tests/Opus/Document/Plugin/IndexTest.php

```diff
@@ -71,8 +68,7 @@ public function testCreateIndexJob()
 
     public function testDoNotCreateIndexJobIfAsyncDisabled()
     {
-
-        Zend_Registry::get('Zend_Config')->runjobs->asynchronous = 0;
+        Zend_Registry::get('Zend_Config')->runjobs->asynchronous = ''; // false
```

Comment:
Das macht mir Sorgen. Ja, Zend_Config liefert, zumindest in ZF1, einen leeren String für *false*. Wenn man das nicht weiß, ist die Zeile hier seltsam.